src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,39 @@
+package qna.domain;
+
+import qna.CannotDeleteException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Answers {
+    private List<Answer> answers;
+
+    private Answers(List<Answer> answers) {
+        this.answers = answers;
+    }
+
+    public static Answers of(Answer answer) {
```

Comment:
```suggestion
    public static Answers of(Answer[] ...answers) {
```
이런 형태의 인자로 받아와서 처리하면 어떨까요?